src/Controller/CardController.php

```diff
@@ -84,4 +86,16 @@ public function showCard(ShowCard $showCard): Response
 
         return $this->json($card, Response::HTTP_OK);
     }
+
+    /**
+     * Edit Card
+     * @Route("/card/{id}",name="edit_card",methods={"PATCH"})
+     */
+    public function editCard(EditCard $editCard): Response
+    {
+        $cardUpdater = new CardUpdateHandler($editCard, $this->entityManager, $this->validator);
```

Comment:
IMHO this instantiation is not correct. `CardUpdateHandler` is the object that contain logic to update entity. If we have class like that then in common we should inject it through constructor. Because that make it easier to maintain our codebase. Eg we could inject different implementation of this handler if something change in logic or if we want to mock it in our tests. But now we can't.